src/ledger/LedgerTxn.cpp

```diff
@@ -2123,6 +2123,9 @@ LedgerTxnRoot::Impl::commitChild(EntryIterator iter, LedgerTxnConsistency cons)
     // std::unique_ptr<...>::swap does not throw
     mHeader.swap(childHeader);
     mChild = nullptr;
+
+    mPrefetchHits = 0;
```

Comment:
I think you missed the same in `LedgerTxnRoot::Impl::rollbackChild`.

Reply:
Good catch, updated.